02-builder-analysis/exercises/zgz-bikes.md

```diff
@@ -0,0 +1,124 @@
+- [Analysis 1: Create areas of influence from bikes anchors](#aoi)
+- [Analysis 2: Intersect areas and city blocks](#intersect)
+- [Analysis 3: Assign total population to city blocks](#do)
+- [Widgets and auto-styling](#widgets)
+- [Publish](#publish)
+
+<hr>
+
+## Create a map <a name="map"></a>
+
+* Click on **`NEW MAP`**.
+* Click on **`CONNECT DATASET`**. Submit this url, choose any `Sync my data` option if sync table feature is available for your account, and make sure the `Let CARTO automatically guess data types and content on import` box is checked: [http://www.zaragoza.es/api/recurso/urbanismo-infraestructuras/estacion-bicicleta.geojson?rf=html&results_only=false&srsname=wgs84&rows=129](http://www.zaragoza.es/api/recurso/urbanismo-infraestructuras/estacion-bicicleta.geojson?rf=html&results_only=false&srsname=wgs84&rows=129). Click **`CONNECT DATASET`** button.
+* Click on **`CREATE MAP`**.
+* From the map's **`LAYERS`** pane click **`ADD`**. Click **`CONNECT DATASET`**. Connect the `Zaragoza blocks` dataset using the same steps as above, with this url: [https://builder-demo.carto.com/api/v2/sql?q=SELECT+*+FROM+manzana&format=shp&filename=manzanas)
+* Double-click on the map layer titles to rename them "Bike anchors" and "Blocks". You can also rename the map this way.
+
+![layers](imgs/zgzbikes/01-layers.png)
+
+
+## Create areas of influence from bikes anchors <a name="aoi"></a>
+
+* Click on "Bike anchors" layer.
+* Go to **`ANALYSIS`** tab.
+* Click on **`ADD ANALYSIS`**.
+* Select `Create areas of influence` analysis and click on **`ADD ANALYSIS`**.
+* Set `RADIUS` to `250` meters and `BOUNDARIES` to `Dissolved`, and click on **`APPLY`**.
+
+![aoi](imgs/zgzbikes/02-aoi.png)
+
+* In order to recover our points layer, go to the main menu and drag the layer source node out.
+
+![aoi](imgs/zgzbikes/03-node.png)
+
+* Rename the other layer to "AOI" to distinguish the areas of influences and the original points.
+
+
+## Intersect areas and city blocks <a name="intersect"></a>
+
+* Click on "Blocks" layer.
+* Click it's **`ANALYSIS`** tab, and click on **`ADD ANALYSIS`**. Select the `Intersect second layer` analysis and click **`ADD ANALYSIS`**.
+* In the `INTERSECT LAYER` field, select the `AOI` node
+* Let the rest of parameters as default.
+* Click on **`APPLY`**.
+
+![intersect](imgs/zgzbikes/04-intersect.png)
+
+* To improve the visualization, we will remove the fill from the "AOIs" layer.
+* Go back to the main menu and select "AOIs" layer.
+* Go to the **`STYLE`** tab, click on the color `FILL` and set the opacity parameter (`A`) to `0`).
+* You can set the `STROKE` color to red.
+
+![style](imgs/zgzbikes/05-style.png)
+
+## Assign total population to city blocks <a name="do"></a>
+
+* Go to the main menu and click on the "Blocks" layer.
+* Click it's **`ANALYSIS`** tab, and click on **`+`** to add a new analysis to the workflow chain. Select the `Enrich with Data Observatory` analysis and click **`ADD ANALYSIS`**.
```

Comment:
LAYERS pane, and be mindful of "it's", which means "it is". Best to just say "the ANALYSIS tab"